### PR TITLE
[docker] added dev flow

### PR DESCRIPTION
**Pull request by mimoo**
I couldn't find an actual flow for building/dev'ing using Docker, and as I'm running into build issues on Mac I made this easy-to-set-up dev workflow.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,3 +76,4 @@ automation/services/watchdog/check_libp2p/check_libp2p
 *.backup
 
 *.terraform.lock.hcl
+_opam
```
Comment (mimoo) [inline .gitignore:79]: local switches

File: src/app/reformat/reformat.ml [hunk 2]
```diff
@@ -16,7 +16,8 @@ let dirs_trustlist =
   ; "tablecloth"
   ; "zexe"
   ; "marlin"
-  ; "snarky" ]
+  ; "snarky" 
+  ; "_opam" ]
```
Comment (mimoo) [inline src/app/reformat/reformat.ml:20]: ignore local switches

**Review by lk86 — APPROVED** [review]

**Comment by mimoo** [comment]
@lk86 I don't think I can merge this myself

**Review by mrmr1993 — APPROVED** [review]
This is fab, thanks @mimoo!

**Pull request merged**